examples/blog-starter/posts/get-blog-posts.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs')
+const path = require('path')
+
+const DIR = path.join(process.cwd(), '/pages/posts/')
+const META = /export\s+const\s+meta\s+=\s+(\{(\n|.)*?\n\})/
```

Comment:
@j0lv3r4 this is not necessary if you can import the file and get the meta from it, like [here](https://github.com/zeit/next-site/blob/master/www/scripts/build-rss.js#L8), because you're adding the file to the entries of webpack, you can use everything that's allowed by a page, even mdx.

Reply:
As we discussed, we'll leave it as it is for now. Using `readFileSync` is better in this case.